packages/flutter_lints/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Updates minimum SDK version to Flutter 3.0.
```

Comment:
Hm, is this change actually changing anything? AFICT, 3.0 was the oldest stable version of Flutter that this package supported even before this change.

Reply:
Oh, is this just to inform people that the package no longer supports pre-releases of 3.0 because the sdk constraints went from 2.17.0-0 to 2.17.0?

Reply:
I forgot this one was weird; I can remove this.

Reply:
I would prefer removing it, it will probably cause more confusion than anything...

Reply:
Done, thanks for catching that.